cni/network/invoker.go

```diff
@@ -0,0 +1,15 @@
+package network
+
+import (
+	"net"
+
+	"github.com/Azure/azure-container-networking/cni"
+	"github.com/Azure/azure-container-networking/network"
+	cniSkel "github.com/containernetworking/cni/pkg/skel"
+	cniTypesCurr "github.com/containernetworking/cni/pkg/types/current"
+)
+
+type IPAMInvoker interface {

```

Comment:
Document your interface? What is deletepoolonerrr? for example? 

Here's a question from left field. Is there a future where we want other cni's invoking cns for ipam? Or would we want them to use the crd?  If we think they might want to hit up cns making this interface really clear might pay future dividedends.

Reply:
Also that first result is ipv4 second is ipv6. (hopefully I die before there is ipv8).